addons/mail/static/src/components/activity/activity.scss

```diff
@@ -172,3 +172,12 @@ a:not([href]):not([tabindex]).o_Activity_detailsButton {
     color: gray('500');
 }
 
+.o_Activity_note, .o_mail_note_title + div {
+  p {
+    margin-bottom: 0;
+  }
+}
```

Comment:
This stylerule is really not nice... kinda a way to say "but don't apply bootstrap margin on <p> in these (broadly) cases".